src/components/InfoPanel.vue

```diff
@@ -218,13 +218,14 @@ export default {
 
       if (selectedLocationType === 'polling-places') {
-        html = 'Day-of voting: Election day is Tuesday, November 3. All polling locations will be open from 6:00 AM to 7:00 PM. As long as you are in line before 7:00 PM, you will still be able to vote. You will need to bring a form of ID to vote in person. More information <a href="https://azsos.gov/elections/voting-election">here</a>. At these locations, you can also drop off the ballot you received by mail.';
+        html = 'All Election Day Voting Locations will be open from 6AM to 7PM on Tuesday, November 3. As long as you are in line before 7PM, you will still be able to vote. Voter identification is required to vote in person. More information <a href="https://azsos.gov/elections/voting-election">here</a>. <hr />At these locations, you can also drop off the ballot you received by mail.';
       } else if (selectedLocationType === 'early-voting-locations') {
-        html = 'Early voting: Early voting in Arizona runs from Tuesday, October 6, to Friday, October 30. Dates and hours vary by location. Voter identification is required to vote in person. More information <a href="https://azsos.gov/elections/voting-election">here</a>. At these locations, you can also drop off the ballot you received by mail.';
+        html = 'Early voting runs until 5PM Friday, October 30. Dates and hours vary by location. Voter identification is required to vote in person. More information <a href="https://azsos.gov/elections/voting-election">here</a>. <hr />At these locations, you can also drop off the ballot you received by mail. Look at Emergency Voting locations for voting options from 5PM Friday, October 30, to 5PM Monday, November 2.';
       } else if (selectedLocationType === 'drop-boxes') {
-        html = 'Ballot drop boxes: You can request a ballot by mail until October 23rd at 5:00 PM by going <a href="https://my.arizona.vote/Early/ApplicationLogin.aspx">here</a>. If you have received a ballot by mail, you can use any drop box in the county of your voter registration up until 7:00 PM on Election Day (Tuesday, November 3). In addition to what you see below, all voting locations when open will accept the ballot you received by mail. If you cannot mail your ballot back by October 27th, it is recommended you drop it off at one of these voting locations or drop boxes.';
+        html = 'If you have received a ballot by mail, you can use any drop box in the county of your voter registration until 7PM Tuesday, November 3 (Election Day). If you cannot mail your ballot back by October 27, it is recommended that you drop it off.
+<hr />In addition to the options below, ALL voting locations when open will accept the ballot you received by mail. Early Voting Locations have options until 5PM Friday, October 30. Emergency Voting Locations have options from 5PM Friday, October 30 to 5PM Monday, November 2. Election Day Voting Locations are all open from 6AM - 7PM on Tuesday, November 3.';
```

Comment:
This newline broke the string, so the compiler complains. 
You can either make it a one long line again, or use backticks, or `\` at the end of line as a break like here:
https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

Reply:
@elizzard very coincidentally I just pulled down `ajlevitz-patch-1` and converted these to template strings so they could be on multiple lines 😄  . I'll push those changes to `master`.

Reply:
:D nice